sympy/plotting/plot.py

```diff
@@ -1843,8 +1853,8 @@ def plot(*args, show=True, **kwargs):
                     'The same variable should be used in all '
                     'univariate expressions being plotted.')
     x = free.pop() if free else Symbol('x')
-    kwargs.setdefault('xlabel', x.name)
-    kwargs.setdefault('ylabel', 'f(%s)' % x.name)
+    kwargs.setdefault('xlabel', latex(x))
+    kwargs.setdefault('ylabel', 'f(%s)' % latex(x))
```

Comment:
Should we set mode='inline'? I think there are some places in the printer that print different things for inline vs. equation

Reply:
That is probably a good idea, but maybe it is not a good idea to try to do this, at least not in this trivial way.

Reply:
Fixed in updated version,